src/utils/__tests__/Navigation.test.js

```diff
@@ -0,0 +1,95 @@
+import Navigation from '../Navigation';
```

Comment:
No notable changes here beyond the change of directory. This file has changed only to match new data structures provided by Astro